elementary/clients/dbt/dbt_runner.py

```diff
@@ -241,3 +242,28 @@ def ls(self, select: Optional[str] = None) -> list:
 
     def source_freshness(self):
         self._run_command(command_args=["source", "freshness"])
+
+    def get_installed_packages(self):
```

Comment:
Care to rename to `get_installed_packages_names` and make it a private method?

Reply:
Sure, I have made the necessary change in the latest commit